src/screens/Library/components/GameCard/index.tsx

```diff
@@ -309,6 +323,11 @@ const GameCard = ({
               { '--installing-effect': installingGrayscale } as CSSProperties
             }
           >
+            {runner === 'legendary' ? (
+              <EpicLogo className="store-icon" />
+            ) : (
+              <GOGLogo className="store-icon" />
+            )}
```

Comment:
maybe show the icon only if there's more than one store logged in? it's kinda useless for people using only one store and it clutters the grid with a repeated icon

Reply:
Yes, makes sense!